docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -4,94 +4,83 @@ By [Phillip Carter](https://github.com/cartermp)
-
-Either toolchain can be used to build and publish libraries today.
-
-The sections of this document dealing with .NET Framework versions or Portable Class Libraries (PCLs) need the .NET Framework installed.  This is only supported on Windows.
-
-To do this, [install the .NET Framework](http://getdotnet.azurewebsites.net).
+The sections of this document dealing with the .NET Framework versions or Portable Class Libraries (PCLs) need the .NET Framework installed.  They are only supported on Windows.  To do this, [install the .NET Framework](http://getdotnet.azurewebsites.net).
 
 Additionally, if you wish to support older targets, you will need to install targeting/developer packs for older framework versions from the [target platforms page](http://getdotnet.azurewebsites.net/target-dotnet-platforms.html).  Refer to this table:
 
-| .NET Framework Version | Thing to download |
+| .NET Framework Version | What to download |
 | ---------------------- | ----------------- |
 | 4.6 | .NET Framework 4.6 Targeting Pack |
 | 4.5.2 | .NET Framework 4.5.2 Developer Pack |
 | 4.5.1 | .NET Framework 4.5.1 Developer Pack |
 | 4.5 | Windows Software Development Kit for Windows 8 |
 | 4.0 | Windows SDK for Windows 7 and .NET Framework 4 |
-| 2.0 and 3.5 | .NET Framework 3.5 SP1 Runtime (or Windows 8+ version) |
+| 2.0, 3.0, and 3.5 | .NET Framework 3.5 SP1 Runtime (or Windows 8+ version) |
 
 ## How to target .NET Core
 
 For the purposes of building a library, targeting ".NET Core" means targeting the .NET Platform Standard.  To see what that means exactly, see [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md).
 
-Note that as of this time, `netstandardX.X` is not supported.  You'll have to use the `dotnet` monikers.  Here's a mapping:
-
-| dotnet | netstandard |
-|--------| ----------- |
-| dotnet51 | netstandard1.0 |
-| dotnet52 | netstandard1.1 |
-| dotnet53 | netstandard1.2 |
-| dotnet54 | netstandard1.3 |
-| dotnet55 | netstandard1.4 |
+The following is what you need to know about .NET Platform Standard targeting:
 
+The version of the .NET Platform Standard you pick will be a tradeoff between access to the newest APIs and ability to target more .NET platforms and Framework versions.  You can do that by picking a version of `netstandardXX` (Where `XX` is a version number) and adding it to your `project.json` file.
 
-The following is what you need to know about .NET Platform Standard targeting:
+Additionally, the corresponding package to depend on is `NETStandard.Library` version `1.5.0-rc2-24027`.  Although there's nothing preventing you from depending on `Microsoft.NETCore.App` like with console apps, it's generally not recommended.  If you need APIs from a package not specified in `NETStandard.Library`, you can always specify that package in addition to `NETStandard.Library` in the `dependencies` section of your `project.json` file.
```

Comment:
here, are you talking about the NuGet package? Maybe say "the corresponding NuGet package to depend on..."
Should you also add a link to https://www.nuget.org/packages/NETStandard.Library/?